package-lock.json

```diff
@@ -33092,7 +33023,7 @@
       "dependencies": {
         "got": {
           "version": "6.7.1",
-          "resolved": "https://registry.npmjs.org/got/-/got-6.7.1.tgz",
+          "resolved": "http://registry.npmjs.org/got/-/got-6.7.1.tgz",
```

Comment:
Could you revert these changes?

Reply:
They happen when I run `npm install`, which I need to do to remove the dependency

Reply:
Let me try an alterantive approach

Reply:
This is just `npm` being `npm` I think.

Here's my env

```
$ npm --version
6.13.4

$ node --version
v12.14.1
```

I ran `npm  uninstall -D static-site-generator-webpack-plugin`

Reply:
Yeah there's no way to stop NPM being dumb, but we can still be selective about what we check in.
It's not a massive issue, but it makes our git history messier if we always let NPM change stuff back and forth

Reply:
I'm weary of manually editing `package-lock.json` though